### PR TITLE
AWS - ACM Integration

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
In Progress

## Related Issues
fixes: demisto/etc#14237

## Description
AWS Certificate Manager Integration

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [x] Code Review

### CODE REVIEW

File: Integrations/AWS-ACM/AWS-ACM.py [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+        data.update({
+            tag['Key']: tag['Value']
+        })
+
+    ec = {'AWS.ACM.Certificates(val.CertificateArn === obj.CertificateArn).Tags': data}
+    human_readable = tableToMarkdown('AWS ACM Certificate Tags', data)
+    return_outputs(human_readable, ec)
+
+
+def get_certificate(args):
```
Comment (yuvalbenshalom) [inline Integrations/AWS-ACM/AWS-ACM.py]: please return the certificate and the certificate chain in 2 separate files and do not print them in an entry or a log.

File: Integrations/AWS-ACM/AWS-ACM.yml [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+  type: 0
+  required: false
+- display: Role Session Duration
+  name: sessionDuration
+  defaultvalue: ""
+  type: 0
+  required: false
+script:
+  script: ''
+  type: python
```
Comment (yuvalbenshalom) [inline Integrations/AWS-ACM/AWS-ACM.yml]: missing subtype

File: Integrations/AWS-ACM/AWS-ACM.yml [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+  commands:
+  - name: aws-acm-describe-certificate
+    arguments:
+    - name: certificateArn
+      required: true
+      description: The Amazon Resource Name (ARN) of the ACM certificate.
+    - name: region
```
Comment (yuvalbenshalom) [inline Integrations/AWS-ACM/AWS-ACM.yml]: please add region selection

File: Integrations/AWS-ACM/AWS-ACM.yml [hunk 4]
```diff
@@ -0,0 +1,393 @@
...
+      description: The value of the tag.
+      type: string
+    description: Lists the tags that have been applied to the ACM certificate. Use
+      the certificate's Amazon Resource Name (ARN) to specify the certificate.
+  - name: aws-acm-get-certificate
+    arguments:
+    - name: certificateArn
+      required: true
+      description: The ARN of the certificate.
+    - name: region
```
Comment (yuvalbenshalom) [inline Integrations/AWS-ACM/AWS-ACM.yml]: please add region selection

File: Integrations/AWS-ACM/AWS-ACM.yml [hunk 5]
```diff
@@ -0,0 +1,393 @@
...
+        from 900 seconds (15 minutes) up to the maximum session duration setting for
+        the role.
+    description: Remove one or more tags from an ACM certificate.
+    execution: true
+  - name: aws-acm-list-tags-for-certificate
+    arguments:
+    - name: certificateArn
+      required: true
+      description: The ARN of the ACM certificate for which you want to list the tags.
+    - name: region
```
Comment (yuvalbenshalom) [inline Integrations/AWS-ACM/AWS-ACM.yml]: please add region selection

File: Integrations/AWS-ACM/AWS-ACM.yml [hunk 6]
```diff
@@ -0,0 +1,393 @@
...
+  - name: aws-acm-remove-tags-from-certificate
+    arguments:
+    - name: certificateArn
+      required: true
+      description: The ARN of the ACM Certificate with one or more tags that you want
+        to remove.
+    - name: tags
+      required: true
+      description: The key-value pair that defines the tag to remove.
+    - name: region
```
Comment (yuvalbenshalom) [inline Integrations/AWS-ACM/AWS-ACM.yml]: please add region selection

File: Integrations/AWS-ACM/AWS-ACM.yml [hunk 7]
```diff
@@ -0,0 +1,393 @@
...
+  - name: aws-acm-add-tags-to-certificate
+    arguments:
+    - name: certificateArn
+      required: true
+      description: String that contains the ARN of the ACM certificate to which the
+        tag is to be applied.
+    - name: tags
+      required: true
+      description: The key-value pair that defines the tag. The tag value is optional.
+    - name: region
```
Comment (yuvalbenshalom) [inline Integrations/AWS-ACM/AWS-ACM.yml]: please add region selection

File: Integrations/AWS-ACM/AWS-ACM.yml [hunk 8]
```diff
@@ -0,0 +1,393 @@
...
+      predefined:
+      - RSA_2048
+      - RSA_1024
+      - RSA_4096
+      - EC_prime256v1
+      - EC_secp384r1
+      - EC_secp521r1
+      description: Specify one or more algorithms that can be used to generate key
+        pairs.
+    - name: region
```
Comment (yuvalbenshalom) [inline Integrations/AWS-ACM/AWS-ACM.yml]: please add region selection

**Comment by yuvalbenshalom** [comment]
CR is done. please prepare docs and Alex review

**Comment by amshamah419** [comment]
@kirbles19 Here are the files for documentation
https://github.com/demisto/content/blob/AWS-ACM/Integrations/AWS-ACM/AWS-ACM.yml
https://github.com/demisto/content/blob/AWS-ACM/Integrations/AWS-ACM/AWS-ACM_desc.md

**Pull request merged**